src/components/utils/header.jsx

```
@@ -1,68 +1,88 @@
+    <header className="inset-x-0 top-0 bg-gradient-to-b from-slate-900 to-black h-min w-full shadow-md select-none z-10">
+      <section className="relative">
+        <img
+          onClick={handleClick}
+          draggable="false"
+          src={Logo}
+          className="transition-all duration-200 ease-in-out cursor-pointer ml-3.5 rounded shadow-md hover:shadow-lg hover:scale-105"
+          alt="logo"
+          width="60"
+          height="60"
+          aria-label="Navigate to home page"
+        />
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)